scripts/cbmc-viewer/reachable.py

```diff
@@ -52,6 +53,17 @@ def __init__(self, pgm="", srcloc=None):
             filename = function['file name']
             filename = srcloc.clean_path(filename)
 
+            # Declare as reachable functions only those functions in
```

Comment:
How would this work for repos that depend on other repos? e.g. `aws-c-io` or the ESDK?

Reply:
It would omit functions not under the source root from the coverage calculation (but obviously not from the checking).

But don't you already use a tree-of-links check to create one source root with all needed repositories under the source root?

Reply:
Sounds good to me then.